frontend/admin/src/js/components/Table/Table.tsx

```diff
@@ -220,20 +310,59 @@ function Table<T extends Record<string, unknown>>({
           </tbody>
         </table>
       </div>
-      {pagination && (
-        <Pagination
-          currentPage={pageIndex + 1}
-          handlePageChange={(pageNumber) => gotoPage(pageNumber - 1)}
-          handlePageSize={setPageSize}
-          pageSize={pageSize}
-          pageSizes={[10, 20, 30, 40, 50]}
-          totalCount={rows.length}
-          ariaLabel={intl.formatMessage({ defaultMessage: "Table results" })}
-          color="black"
-          mode="outline"
-          data-h2-margin="b(all, none)"
-        />
-      )}
+      <div
+        data-h2-align-items="b(center)"
+        data-h2-display="b(flex)"
+        data-h2-bg-color="b(lightgray)"
+        data-h2-justify-content="b(space-between)"
+        data-h2-radius="b(none, none, s, s)"
+        data-h2-padding="b(all, s)"
+      >
+        <div
+          data-h2-display="b(flex)"
+          data-h2-align-items="b(center)"
+          data-h2-margin="b(right, s)"
+        >
+          <p>
+            {intl.formatMessage({
+              defaultMessage: "Selected actions:",
+              description: "Label for action buttons in footer of admin table.",
+            })}
+          </p>
+          <Spacer>
+            <Button type="button" mode="solid" color="primary">
+              {intl.formatMessage({
+                defaultMessage: "Download XML",
+                description:
+                  "Text label for button to download a csv of items in a table.",
```

Comment:
message says xml, description says csv

Reply:
Good catch! Fixed with [update i18n description for xml download](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3114/commits/334ac437532223bbb14e5e61f9b787526dbf8883)